### DISCUSSION TITLE
Wii remote not pairing

### DISCUSSION THREAD

**Discussion by killerdude123** [ndeadly/MissionControl#248] [Q&A]
Hi, I installed the 0.5.0 alpha release on HOS 12.0.1 and got my DS4 working with my switch but I saw on the readme saying it supports wii motes with the tatacon addon so I figured id get it to try but before that I wanted to pair with my wii mote first and here is the problem. I've pressed the red sync button at the back of the remote and the led flashes but my switch doesn't detect the wii mote. Been constantly putting it into sync mode and switch still doesn't detect. Any help would be appreciated

**Comment by ndeadly** [comment]
Yes, this is a known issue. See https://github.com/ndeadly/MissionControl/issues/239#issuecomment-824957678

12.0.0 broke pairing for most controllers. Since I thought I would have to make a new release to fix this anyway after reversing the changes from 12.0.0, I didn't bother updating the part that handles pairing Wii controllers. Turns out the problem was on Nintendo's end, and they fixed it in 12.0.1. Wii controllers will be working again by the time I make the final release. For now you can downgrade below 12.0.0 to pair then upgrade back as a workaround.

**Reply by killerdude123** [reply]
ah i see thanks :)
